Sources/Plasma/Apps/plClient/Mac-Cocoa/PLSView.mm

```diff
@@ -63,39 +62,16 @@
 @interface PLSView ()
 
 @property NSTrackingArea* mouseTrackingArea;
-#if PLASMA_PIPELINE_METAL
-@property(weak) CAMetalLayer* metalLayer;
-#endif
 
 @end
 
 @implementation PLSView
 
-// MARK: View setup
-- (id)initWithFrame:(NSRect)frameRect
-{
-    self = [super initWithFrame:frameRect];
-#if PLASMA_PIPELINE_METAL
-    CAMetalLayer* layer = [CAMetalLayer layer];
-    layer.contentsScale = [[NSScreen mainScreen] backingScaleFactor];
-    layer.maximumDrawableCount = 3;
-    layer.pixelFormat = MTLPixelFormatBGR10A2Unorm;
```

Comment:
Previously we've talked about inspecting the screen bit depth and setting the layer pixel format based on that. I.E. don't set a 10 bit color depth if the screen isn't 10 bit. I don't know if that change is moving ahead - but moving that out of the client would complicate things. The answer might need to be some sort of engine level flag for 10 bit color. In theory - Direct3D 9 supports 10 bit color output as well.

I've also seen Apple just set this to 16 bit in their examples (which is what our fragment shaders output in Metal) and call it a day. But memory usage would be higher.